src/app/login-page/components/login-page/login-page.component.html

```diff
@@ -1,9 +1,106 @@
+                <li class="second_record">
+                  <p class="fd">01</p>
+                  <p class="phone">#98489456<br>
+                    <small class="f_ip">01.01.0001</small>
+                  </p>
+                  <p class="s_ip"><small>02.01.0001</small></p>
+                  <p class="ld"><small>34</small></p>
+                </li>
+                <li class="third_record">
+                  <p class="fd">13</p>
+                  <p class="phone">#98374598743<br>
+                    <small class="f_ip">01.01.0001</small>
+                  </p>
+                  <p class="s_ip"><small>02.01.0001</small></p>
+                  <p class="ld"><small>34</small></p>
+                </li>
+                <li class="fourth_record">
+                  <p class="fd">99</p>
+                  <p class="phone">#825163<br>
+                    <small class="f_ip">01.01.0001</small>
+                  </p>
+                  <p class="s_ip"><small>02.01.0001</small></p>
+                  <p class="ld"><small>34</small></p>
+                </li>
+                <li class="fifth_record">
+                  <p class="fd">53</p>
+                  <p class="phone">#0984356<br>
+                    <small class="f_ip">01.01.0001</small>
+                  </p>
+                  <p class="s_ip"><small>02.01.0001</small></p>
+                  <p class="ld"><small>34</small></p>
+                </li>
+              </ul>
+              <div class="weitere_button">
+                <button type="submit" class="">Weitere gelöste Fälle ansehen</button>
+              </div>
+            </div>
+            <div class="unsere_partner_section">
+              <h3>Unsere Partner</h3>
+              <div class="social_image">
+                <img src="assets/logo/social_image.png" width="100%">
+              </div>
+              <div class="paragraph">
+                <h3>Rechtliches</h3>
+                <p>
+                  Durch die Anmeldung oder Registrierung stimmust du unseren <span class="orange">Nutzungsbedingungen</span> zu. In unserer <span class="orange">Datenrichtlinie</span> erfahrst du, wie wir deine Daten erfassen, verwenden und teilen. Unsere <span class="orange">Cookie-Richtlinie</span> erklart, wie wir Cookies und ahnliche Technologien verwenden.
+                </p>
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                </p>
+              </div>
+            </div>
+          </div>
+      </div>
     </div>
-    
-    <app-login-form></app-login-form>
```

Comment:
You should use a separate component, to store the login form. 
you can use the login-form component, 
and make changes to it as needed. 

put all the scss  in login-form.component.scss